ci/appveyor.yml

```diff
@@ -0,0 +1,78 @@
+# Doesn't work because _miriad doesn't compile on Windows.
```

Comment:
appveyor is the service to do continuous integration testing on a windows platform. I'd take this file out unless you really want to go this route. Note that pyuvdata currently doesn't work on windows.